src/assets/scripts/client/trafficGenerator/SpawnPatternModel.js

```diff
@@ -770,13 +770,21 @@ export default class SpawnPatternModel extends BaseModel {
             this.cycleStartTime += this.period;
 
             return TIME.ONE_HOUR_IN_SECONDS / (this.rate + (progressInPeriod - 4) * this.variation);
-        } else if (progressInPeriod <= 1) {
```

Comment:
I saw this a few times already, why the move to `if` instead of the `else if`?  is the linter really complaining about this?